### PR TITLE
Test non-lint failure

### PR DESCRIPTION
**Pull request by erik-whiting**
Let's see what a non-linting failure looks like

### CODE REVIEW

**Comment by erik-whiting** [comment]
huh

File: tests/test_integrity.py [hunk 1]
```diff
@@ -333,7 +333,7 @@ def test_repository_license(self):
                 self.assertEqual(
                     spdx,
                     OBO_TO_SPDX[obo_license],
-                    msg="OBO Foundry license annotation does not match GitHub license",
+                    msg="FORCING FAILURE",
```
Comment (erik-whiting) [inline tests/test_integrity.py:336]: so this is testing nothing because the build passed with this

**Comment by nlharris** [comment]
Can this be closed or merged?

**Comment by erik-whiting** [comment]
this was a test branch that should be closed, i'll close it

**Pull request closed without merging**